README.md

```diff
@@ -122,28 +122,27 @@ sentences from one open source project may be used to train the sentiment model
 to automatically analyze another project.
 
 The Stanford CoreNLP sentiment models are trained on movie reviews and aren't
-very good for analyzing sentiment of code reviews. It tends to look at the
-sentence structure of technical comments and rank it as a negative tone, even
-if there are no negative words. It's also not trained for curse words or
+very good for analyzing sentiment of code reviews. They tend to look at the
+sentence structure of technical comments and rank them as a negative tone, even
+if there are no negative words. They're also not trained for curse words or
 emojis.
 
 The Stanford CoreNLP includes a way to retrain the neural network to recognize
 sentiment of sentence structures. You have to feed it a training set (their
-training set is ~8,000 sentences) and a development set that helps you tune
+training set is ~8,000 sentences) and a development set that helps you tunne1
```

Comment:
Aha, I think a stray 1 ended in there by accident, I saw "tune1," and turned it to tunnel. :)